FAQ.md

```diff
@@ -0,0 +1,13 @@
+<b>Frequently Asked Questions</b>
```

Comment:
Just answer these questions about the tool:
1. What is Book Uploader Bot (BUB2)?
2. What does this tool do?
3. Who can benefit from this?
4. Future enhancements?

Reply:
Sure.Please answer what are future enhancements?

Reply:
I have answered above all questions except Future Enhancements question is left.If that is answered i'll send a final commit.

Reply:
> I have answered above all questions except Future Enhancements question is left.If that is answered i'll send a final commit.

Future enhancements:
Bulk upload future, MediaWiki OAuth integration, Direct uploading to Commons